### PR TITLE
Keep source directory structure.

### PR DESCRIPTION
**Pull request by mkubenka**
Follow up to #3 with fix to ensure all files are written to `outputDir`.

### CODE REVIEW

File: bin/shuji.js [hunk 1]
```diff
@@ -163,28 +163,24 @@ fileList.forEach((filepath) => {
...
+    const safeSuffix = path.normalize(item).replace(/^(\.\.[/\\])+/, '');
+    const outfile = path.resolve(outputDir, safeSuffix);
+
+    fs.ensureDirSync(path.dirname(outfile));
 
     if (opts.verbose) {
       console.log(`Writing to file ${outfile}`);
     }
 
-    if (fs.existsSync(outfile)) {
```
Comment (paazmaya) [inline bin/shuji.js]: Is there a reason why there should not be a complaiment when the given file already exists?

Comment (verdverm) [reply bin/shuji.js]: What would you like to see here?

I just tried this PR out and iiiiiiiit's gRRRReat !  Would love to see it merged pending this change

Comment (mkubenka) [reply bin/shuji.js]: @paazmaya maybe adding new `--force` option to overwrite existing files?

Comment (paazmaya) [reply bin/shuji.js]: `--force` or `--overwrite` would be great

**Comment by sijad** [comment]
any news for this one?

**Comment by paazmaya** [comment]
The comments have not yet been addressed..

**Comment by jingsam** [comment]
@sijad you can try `npm install @jingsam/shuni`

**Comment by falconmick** [comment]
Very interested in this!

**Pull request closed without merging**